kvm-install-vm

```diff
@@ -73,12 +73,12 @@ function usage_subcommand ()
             printf "    centos7-atomic  CentOS 7 Atomic Host                centos\n"
             printf "    centos6         CentOS 6                            centos\n"
             printf "    debian9         Debian 9 (Stretch)                  debian\n"
-            printf "    fedora27        Fedora 27                           fedora\n"
-            printf "    fedora27-atomic Fedora 27 Atomic Host               fedora\n"
-            printf "    fedora28        Fedora 28                           fedora\n"
-            printf "    fedora28-atomic Fedora 28 Atomic Host               fedora\n"
+            printf "    fedora29        Fedora 29                           fedora\n"
+            printf "    fedora29-atomic Fedora 29 Atomic Host               fedora\n"
+            printf "    fedora30        Fedora 30                           fedora\n"
             printf "    ubuntu1604      Ubuntu 16.04 LTS (Xenial Xerus)     ubuntu\n"
             printf "    ubuntu1804      Ubuntu 18.04 LTS (Bionic Beaver)    ubuntu\n"
+            printf "    ubuntu1904      Ubuntu 19.04 LTS (Disco Dingo)      ubuntu\n"
```

Comment:
Ubuntu 19.04 is not LTS, is it?

Reply:
You are correct, it is not LTS. I saw the 19.04 and incorrectly assumed LTS because of the .04. 20.04 will be the next LTS.